eng/targets/Helix.props

```diff
@@ -15,14 +15,22 @@
     <RunQuarantinedTests>false</RunQuarantinedTests>
     <IsWindowsHelixQueue>false</IsWindowsHelixQueue>
     <IsWindowsHelixQueue Condition="$(HelixTargetQueue.Contains('Windows')) or $(HelixTargetQueue.Contains('windows'))">true</IsWindowsHelixQueue>
-    <_HelixRunType Condition="'$(RunQuarantinedTests)' == 'true'">quarantined</_HelixRunType>
-    <_HelixRunType Condition="'$(RunQuarantinedTests)' != 'true'">unquarantined</_HelixRunType>    
-    <HelixTestName>$(MSBuildProjectName)--$(TargetFramework)--$(_HelixRunType)</HelixTestName>
+    <HelixTestName>$(MSBuildProjectName)--$(TargetFramework)</HelixTestName>
```

Comment:
nit: If this affects how results display in the Tests tab, suggest keeping this here _too_. That'll make it possible to grok a single (failed) PR build pretty easily.

Reply:
It doesn't really. It would affect the Helix "work item" entry but I don't know that it would affect the test entry itself.